qcodes/instrument_drivers/Aviosys/IP_Power_9258S.py

```diff
@@ -0,0 +1,92 @@
+from qcodes import Instrument, InstrumentChannel, ChannelList
+from qcodes.utils.validators import Enum
+from qcodes.utils.helpers import create_on_off_val_mapping
+import urllib
+
+
+class PowerChannel(InstrumentChannel):
+    """
+    Channel class for a socket on the Aviosys IP Power 9258S.
+    Args:
+        parent (Instrument): Parent instrument.
+        name (str): Channel name.
+        channel (str): Alphabetic channel id.
+    Attributes:
+        channel (str): Alphabetic channel id.
+        channel_id (int): Numeric channel id.
+    """
+
+    _channel_values = Enum('A', 'B', 'C', 'D')
+    _channel_ids = {'A': 1, 'B': 2, 'C': 3, 'D': 4}
+
+    def __init__(self, parent, name, channel):
+
+        super().__init__(parent, name)
+
+        # validate the channel id
+        self._channel_values.validate(channel)
+        self.channel = channel
+        self.channel_id = self._channel_ids[channel]
+
+        # add parameters
+        self.add_parameter('power',
+                           get_cmd=self.get_power,
+                           set_cmd=self.set_power,
+                           get_parser=int,
+                           val_mapping=create_on_off_val_mapping(on_val=1, off_val=0),
+                           label='power {}'.format(self.channel))
+
+    # get methods
+    def get_power(self):
```

Comment:
since the users are expected to interact with parameters, and not with such methods directly, it makes sense to make these get/set methods (that are used in parameters) private, by prefixing their names with `_`. Would you mind doing this for "the get/set methods that are used in parameters" in this PR?